crates/rosty/src/node.rs

```diff
@@ -172,6 +201,17 @@ impl Node {
         }
     }
 
+    /// Returns true if this node is using the simulated time
+    pub fn is_using_sim_time(&self) -> bool {
+        self.sim_time.is_some()
+    }
+
+    /// Returns the last simulated clock if it is available
+    pub fn get_last_sim_clock(&self) -> Option<Duration> {
+        let duration = self.sim_time.as_ref()?.duration();
```

Comment:
Remove let duration? is already clear from the call to duration

Reply:
Done